test/foundry/EspressoTEEVerifier.t.sol

```diff
@@ -3,62 +3,121 @@ pragma solidity ^0.8.20;
 
 import "forge-std/Test.sol";
 import {EspressoTEEVerifier} from "../../src/bridge/EspressoTEEVerifier.sol";
-import {
-    TransparentUpgradeableProxy
-} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
-import {
-    AutomataDcapAttestation
-} from "@automata-network/dcap-attestation/contracts/AutomataDcapAttestation.sol";
-import {PCCSRouter} from "@automata-network/dcap-attestation/contracts/PCCSRouter.sol";
-
-import {PCCSSetupBase} from "@automata-network/dcap-attestation/test/utils/PCCSSetupBase.sol";
-import {RiscZeroSetup} from "@automata-network/dcap-attestation/test/utils/RiscZeroSetup.sol";
-
-contract EspressoTEEVerifierTest is Test, PCCSSetupBase, RiscZeroSetup {
+
+contract EspressoTEEVerifierTest is Test {
     address proxyAdmin = address(140);
     address adminTEE = address(141);
     address fakeAddress = address(145);
 
     EspressoTEEVerifier espressoTEEVerifier;
-    PCCSRouter pccsRouter;
-    bytes32 imageId = vm.envBytes32("DCAP_IMAGE_ID");
-
-    function setUp() public override {
-        super.setUp();
-
-        // PCCS Setup
-        pccsRouter = setupPccsRouter();
-        pcsDaoUpserts();
-        espressoTEEVerifier = new EspressoTEEVerifier(address(pccsRouter));
+    bytes32 reportDataHash =
+        bytes32(0x739f5f48d929cc121c080ec6527a22be3c69bad5c40606cd098a9fa7ed971f1b);
+    bytes32 mrEnclave = bytes32(0x51dfe95acffa8a4075b716257c836895af9202a5fd56c8c2208dacb79c659ff0);
+    bytes32 mrSigner = bytes32(0x0c8242bba090f54b10de0c2d1ca4b633b9c08b7178451c71d737c214b72fc836);
 
-        string memory tcbInfoPath = "/test/foundry/configs/tcbinfo.json";
-        string memory qeIdPath = "/test/foundry/configs/tee_identity.json";
-        qeIdDaoUpsert(3, qeIdPath);
-        fmspcTcbDaoUpsert(tcbInfoPath);
+    function setUp() public {
+        vm.createSelectFork("https://rpc.ankr.com/eth_sepolia");
```

Comment:
I am forking sepolia in tests here instead of arb sepolia because automata hasnt deployed there contracts on arb sepolia yet. I will migrate these tests once they have done that